src/http.c

```diff
@@ -90,7 +90,7 @@ int headers_complete_cb(http_parser *p)
   // Forwarded: by=<identifier>; for=<identifier>; host=<host>; proto=<http|https>
   char *proto = context->https ? "https" : "http";
   strcpy(request->headers[request->num_headers][0], "Forwarded");
```

Comment:
isn't it keyword `X-Forwarded-For`?

Reply:
check this out https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For and https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Proto as well

Reply:
As far as I understand `X-Forwarded-For` is not standardized.
> A standardized version of this header is the HTTP Forwarded header.
[Source](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For)
[Forvarded header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded)

Reply:
none of them are standard headers, I just want it to work so please do it like that. thanks :)